Update README.md of github organization page

Sakrecoer opened the pull request.

Sakrecoer committed:
Update README.md of github organization page

Sakrecoer committed:
Update README.md

Sakrecoer committed:
Update README.md

Sakrecoer committed:
remove eroneous banner

Sakrecoer committed:
Make banner a link to be github sponsor

Sakrecoer committed:
typo

puria commented on profile/README.md:
```diff
@@ -1,11 +1,14 @@
+🖖 Welcome, fellow dyne! 
 
-We have more than **20 years of expertise** putting computers at work to **empower people**.
+[Dyne.org](https://dyne.org) is a non-profit **free software foundry** and a merry **hacker community**.<br>We have nearly **30 years of expertise** putting computers at work to **empower people**.
```
avoid the `<br/>` if possibile you can achive that with two newlines in markdown

jaromil replied:
wells potted, seems good also to keep on same line

Sakrecoer replied:
2 news lines creates 2 new lines :/ (why, mr Markdown? Please explain yourself!)  
Github markdown uses `<br>` wildly and widely. But sure, 2 lines it is!

puria commented on profile/README.md:
```diff
@@ -1,11 +1,14 @@
+
+**📢 Get news from us on [our blog](https://news.dyne.org) and tune in to our corner of the [fediverse](https://fed.dyne.org)**
+
+[![Github organization banner](https://github.com/Sakrecoer/.github-1/assets/3484593/7ada54a5-95e3-4f3d-8bb3-17101790a4ae)](https://github.com/sponsors/dyne)
```
mayb you want to add the asset directly in this repo in place of the `Sakrecoer/.github-1`

Sakrecoer replied:
yes! i didn't realize, thank you. Problem is i don't have access to the https://github.com/dyne/.github

Sakrecoer committed:
address review

- no more `<br>`
- banner hosted on dyne.org
- section for endorsements and supporters
